lib/payment_processor/go_cardless/populator.rb

```diff
@@ -69,6 +77,16 @@ def client
         )
       end
 
+      def subscription_start_date
+        date = Date.today.change(day:20)
+
+        if Time.now.day >= settings_charge_day
+          date += 1.month
+        end
+
+        date
+      end
+
       def charge_date
         if Settings.gocardless.gbp_charge_day.blank? || mandate.scheme.downcase != 'bacs'
```

Comment:
since you introduced `bacs?`, could you use it here as well: `if Settings.gocardless.gbp_charge_day.blank || !bacs?